pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -241,7 +241,7 @@ func (m SQSMonitor) processInterruptionEvents(interruptionEventWrappers []Interr
 	}
 
 	if failedInterruptionEventsCount != 0 {
-		return fmt.Errorf("some interruption events for message Id %b could not be processed", message.MessageId)
+		return fmt.Errorf("some interruption events for message Id %s could not be processed", *message.MessageId)
```

Comment:
Thanks for fixing this!

One of the [unit tests is failing](https://github.com/aws/aws-node-termination-handler/runs/8215609377?check_suite_focus=true#step:5:578) because our mock SQS messages don't have a `MessageId` property in them. Can you add it? Replace the last line of `getSQSMessageFromEvent()` of [`pkg/monitor/sqsevent/sqs-monitor_test.go`](https://github.com/aws/aws-node-termination-handler/blob/31f4b9f2e2f2e1fd35d558f14e4e97673acebee8/pkg/monitor/sqsevent/sqs-monitor_test.go#L915-L922) with something like this:

```
messageId := "1234567890"
return sqs.Message{Body: &eventStr, MessageId: &messageId}, nil
```